src/Moryx.Products.Management/Facades/ProductManagementFacade.cs

```diff
@@ -43,6 +48,42 @@ public void Deactivate()
 
         public string Name => ModuleController.ModuleName;
 
+        public IDictionary<string, object> Importers
+        {
+            get
+            {
+                ValidateHealthState();
+                return ProductManager.Importers.ToDictionary(i => i.Name, i => i.Parameters);
```

Comment:
I propose to delegate this to the Storage. It already keeps a dictionary of configured types. Just needs an additional internal API or sth